.github/workflows/apk-release.yml

```diff
@@ -15,6 +15,7 @@ env:
 
 jobs:
   release:
+    if: false
```

Comment:
**NB:** Disabling this since we no longer use it. Might be refactored and used later though for CLI triggered releases. Disabling prevents inadvertent triggering.